pkg/manager/member/startscript/v2/common.go

```diff
@@ -37,6 +37,65 @@ then
     echo "entering debug mode."
     tail -f /dev/null
 fi
+`
+	dnsAwaitPart = "<<dns-await-part>>"
+
+	componentCommonWaitForDnsIpMatchScript = `
+elapseTime=0
+period=1
+while true; do
+    sleep ${period}
+    elapseTime=$(( elapseTime+period ))
+
+    if [[ ${elapseTime} -ge ${waitThreshold} ]]; then
+        echo "waiting for pd cluster ready timeout" >&2
```

Comment:
`pd` is not `common` as this script is used both for PD and TiKV.

Reply:
fixed